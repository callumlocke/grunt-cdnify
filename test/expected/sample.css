body {
  background-image: url('//cdn.example.com/stuff/foo.jpg');
  background-image: url('//cdn.example.com/foo.jpg');
  background-image: url('//cdn.example.com/foo/bar/test.jpg');
  background-image: url('//cdn.example.com/foo/bar/test.jpg');
  background-image: url('//cdn.example.com/foo.jpg');
  background-image: url('//cdn.example.com/stuff/foo/bar.jpg');
}
