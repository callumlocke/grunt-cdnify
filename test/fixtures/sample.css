body {
  background-image: url('foo.jpg');
  background-image: url('/foo.jpg');
  background-image: url('/foo/bar/test.jpg');
  background-image: url('../../foo/bar/test.jpg');
  background-image: url('../foo.jpg');
  background-image: url('foo/bar.jpg');
}
