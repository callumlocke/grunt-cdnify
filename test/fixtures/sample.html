<html>
  <head>
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,400italic,700">
    <link rel="icon" href="/favicon.ico">
    <link rel="apple-touch-icon" href="/apple-touch-icon.png">
    <style>
      body {
        background-image: url('foo.jpg');
        background-image: url('/foo.jpg');
        background-image: url('../foo.jpg');
        background-image: url('foo/bar.jpg');
      }
    </style>
  </head>

  <body>
    <h1>Hello</h1>
    <img something src="bar.jpg" data-hiya data-food="crimson">
    <img something data-src="bar.jpg" src="foo.jpg" data-hiya data-food="crimson">

    <img src=foo/lala.png foo=bar />
    <img truffles=abc.jpg>
    <img ng-src=foo/lala.png/>

    <img src="blah.png" something=blah  > :)

    <div style="background-image:url('/images/homePage/fe18b3f3f137.1.png')"></div>

    <script src="hello.js"></script>
  </body>
</html>
