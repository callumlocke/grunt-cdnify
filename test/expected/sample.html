<html>
  <head>
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,400italic,700">
    <link rel="icon" href="//cdn.example.com/favicon.ico">
    <link rel="apple-touch-icon" href="//cdn.example.com/apple-touch-icon.png">
    <style>
      body {
        background-image: url('//cdn.example.com/stuff/foo.jpg');
        background-image: url('//cdn.example.com/foo.jpg');
        background-image: url('//cdn.example.com/foo.jpg');
        background-image: url('//cdn.example.com/stuff/foo/bar.jpg');
      }
    </style>
  </head>

  <body>
    <h1>Hello</h1>
    <img something src="//cdn.example.com/stuff/bar.jpg" data-hiya data-food="crimson">
    <img something data-src="//cdn.example.com/stuff/bar.jpg" src="//cdn.example.com/stuff/foo.jpg" data-hiya data-food="crimson">

    <img src=//cdn.example.com/stuff/foo/lala.png foo=bar />
    <img truffles=abc.jpg>
    <img ng-src=foo/lala.png/>

    <img src="//cdn.example.com/stuff/blah.png" something=blah  > :)

    <div style="background-image:url('//cdn.example.com/images/homePage/fe18b3f3f137.1.png')"></div>

    <script src="//cdn.example.com/stuff/hello.js"></script>
  </body>
</html>
